src/main/java/edu/wpi/grip/ui/ExceptionView.java

```diff
@@ -0,0 +1,190 @@
+package edu.wpi.grip.ui;
+
+import javafx.application.HostServices;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.scene.Parent;
+import javafx.scene.control.*;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.layout.GridPane;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Displays an alert with an exception and provides a place for a user to supply information about what caused the error.
+ * Includes a textbox with formatted markdown to allow the issue to be pasted into GitHub easily.
+ * Also, provides links with quick access to the githup issue page.
+ */
+public final class ExceptionView extends Alert {
```

Comment:
`ExceptionDialog` or `ExceptionAlert` would be more appropriate I think, since this isn't a JavaFX control